bioio_dv/reader.py

```diff
@@ -50,7 +50,11 @@ class Reader(reader.Reader):
 
     @staticmethod
     def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
-        return DVFile.is_supported_file(path)
+        if not DVFile.is_supported_file(path):
```

Comment:
This is the same logic as one of the other [PR](https://github.com/bioio-devs/bioio-nd2/pull/19)s but in a different order. We should try to be consistent